dive_core/shader_disassembly.cpp

```diff
@@ -19,12 +19,14 @@
 #include "dive_core/common/common.h"
 #include "pm4_info.h"
 
-#ifndef _MSC_VER
+#ifdef _MSC_VER
```

Comment:
I was following the compile guards which previously specified _MSC_VER since that is what's being used to compile everything versus _WIN32 specifying the platform.